variables.tf

```diff
@@ -15,9 +15,136 @@
  */
 
 variable "project_id" {
-  description = "The project ID to deploy to"
+  description = "The project ID used for the Datalab instance"
 }
 
-variable "bucket_name" {
-  description = "The name of the bucket to create"
+variable "name" {
+  description = "The name of the Datalab instance"
+  default     = "datalab"
+}
+
+variable "zone" {
+  description = "The zone the Datalab instance will be deployed to"
+}
+
+variable "network_name" {
+  description = "The network the Datalab instance will be in"
+}
+
+variable "subnet_name" {
+  description = "The subnet the Datalab instance will be in"
+}
+
+variable "service_account" {
+  description = "The service account attached to the Datalab instance. If empty the default Google Compute Engine service account is used"
+  default     = ""
```

Comment:
With Terraform 0.12, we can use `null` rather than an empty string to signify an unspecified value.